app/views/teams/users/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1 class="usa-display">Add New user</h1>
```

Comment:
**suggestion:** can we add the team name here? Otherwise it can be confusing / users have issues remembering the context of which team they're adding for.